pkg/cache/context.go

```diff
@@ -606,6 +606,17 @@ func (ctx *Context) PublishEvents(eventRecords []*si.EventRecord) {
 						zap.String("taskID", taskID),
 						zap.String("event", record.String()))
 				}
+			case si.EventRecord_NODE:
+				nodeID := record.ObjectID
+				node := ctx.schedulerCache.GetNode(nodeID).Node()
```

Comment:
we need to handle the case where `ctx.schedulerCache.GetNode` could be nil.
once this is fixed, I think this can be merged.

Reply:
Indeed. Fixed